packages/gdl-frontend/components/WelcomeTutorial/WelcomeTutorial.js

```diff
@@ -199,120 +186,121 @@ class WelcomeTutorial extends React.Component<Props, State> {
+              this.handleTouchEnd(isTablet);
+            }}
           >
-            <div
-              style={{
-                display: this.state.isHidden1,
-                left: this.state.left + 'px',
-                position: 'relative'
-              }}
+            <Slide
+              style={{ height: '100%' }}
+              direction={this.state.direction}
+              in={this.state.activeStep === 0}
             >
-              <div>
+              <div
+                style={{
+                  display: this.state.activeStep === 0 ? 'flex' : 'none',
+                  left: this.state.left,
+                  position: 'relative'
+                }}
+              >
                 <GetDialogContent
                   screenshotUrlM={selectLanguage}
                   screenshotUrl={selectLanguage}
                   graceImgUrl={grace1}
                   message="Hi! If you want to change the language, click on the globe at the top-right corner of the screen!"
-                  fullscreen={mobile}
+                  fullscreen={isTablet}
                 />
               </div>
-            </div>
-          </Slide>
-          <Slide
-            style={{ height: '100%' }}
-            direction={this.state.direction}
-            in={this.state.activeStep === 1}
-          >
-            <div
-              style={{
-                display: this.state.isHidden2,
-                left: this.state.left + 'px',
-                position: 'relative'
-              }}
+            </Slide>
+            <Slide
+              style={{ height: '100%' }}
+              direction={this.state.direction}
+              in={this.state.activeStep === 1}
             >
-              <GetDialogContent
-                screenshotUrlM={saveOfflineM}
-                screenshotUrl={saveOffline}
-                graceImgUrl={grace2}
-                message="You can save books for later and read them offline by clicking the Save offline icon!!."
-                fullscreen={mobile}
-              />
-            </div>
-          </Slide>
-          <Slide
-            style={{ height: '100%' }}
-            direction={this.state.direction}
-            in={this.state.activeStep === 2}
-          >
-            <div
-              style={{
-                display: this.state.isHidden3,
-                left: this.state.left + 'px',
-                position: 'relative'
-              }}
+              <div
+                style={{
+                  display: this.state.activeStep === 1 ? 'flex' : 'none',
+                  left: this.state.left,
+                  position: 'relative'
+                }}
+              >
+                <GetDialogContent
+                  screenshotUrlM={menuM}
+                  screenshotUrl={menu}
+                  graceImgUrl={grace3}
+                  message="The menu is in the top-left corner. From the menu you can access Categories, Favorites and more."
```

Comment:
Denne ønsker vi på sikt å kunne få oversatt til forskjellige språk. Som du sikkert har sett noen steder bruker vi da noe som heter `react-intl` for å definere `id'er` som man kan slå opp på for å få oversatt tekststreng. Du kan se et oversiktlig eksempel på det i `NoResults.js` :)